app/api/v2/oauth.py

```diff
@@ -66,11 +66,11 @@ async def token(
     grant_type: str = Form(),
     client_id: int = Form(default=None),
     client_secret: str = Form(default=None),
-    auth_credentials: Optional[dict[str, Union[str, int]]] = Depends(
+    auth_credentials: Optional[dict[str, Any]] = Depends(
```

Comment:
@alowave223 typing the possible values of a dict is usually a bad idea imo - if you're going to try, consider using a [typing.TypedDict](https://docs.python.org/3/library/typing.html#typing.TypedDict)